webpack.common.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const WebpackNotifierPlugin = require("webpack-notifier");
+const webpack = require("webpack");
+const path = require("path");
+const absolute = (relPath) => path.join(__dirname, relPath);
+const EnvLoaderPlugin = require("webpack-env-loader-plugin");
+
+let env = process.env.NODE_ENV || "development";
```

Comment:
This is now duplicated below in `webpack.config.js:11`. Not sure if we should possibly just `module.exports` this value as well so everything stays in sync.

Can't think of any other clever ideas, but it looks like we're also using `env === "development"`, `env !== 'test'`, and `env === 'production'` in these files. Probably overkill to export `isDevelopmentEnv()`, `isTestEnv()`, and `isProductionEnv()` methods.